@here/harp-mapview/lib/EventDispatcher.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as THREE from "three";
+
+/**
+ * JavaScript events for custom objects. Stores all listeners to allow removing all listeners for
+ * housekeeping.
+ *
+ */
+export class EventDispatcher {
```

Comment:
Please add a note that we will switch back to `THREE.EventDispatcher` once https://github.com/mrdoob/three.js/pull/19844 is released

Reply:
will do.